src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -1096,5 +1096,40 @@ void ClusterTaskManager::SpillWaitingTasks() {
   }
 }
 
+ResourceSet ClusterTaskManager::CalcNormalTaskResources() const {
+  TaskRequest total_normal_task_request;
```

Comment:
```suggestion
  std::unordered_map<std::string, FixedPoint> total_normal_task_request;
```

Reply:
Yes, this looks better. In this way, the above "operator+=" and TaskRequestToResourceMap() are no longer needed as well.